src/lib/Zikula/Core/Forms/ZikulaCsrfProvider.php

```diff
@@ -11,12 +11,12 @@
 
 namespace Zikula\Core\Forms;
 
-use Symfony\Component\Form\Extension\Csrf\CsrfProvider\CsrfProviderInterface;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
 
 /**
  * Csrf provider based on zikula csrf system.
  */
-class ZikulaCsrfProvider implements CsrfProviderInterface
+class ZikulaCsrfProvider implements CsrfTokenManagerInterface
```

Comment:
I would be curious to know if the `ZikulaCsrfProvider` is even necessary at all. Why aren't we using the native one?